client/js/Ui/searchpanel.js

```diff
@@ -18,72 +46,45 @@ const searchByIdentifier = (identifier) => {
 SCWeb.ui.SearchPanel = {
     init: function () {
         return new Promise(resolve => {
-            var keynode_nrel_main_idtf = null;
-            var keynode_nrel_idtf = null;
-            var keynode_nrel_system_idtf = null;
-
             $('.typeahead').typeahead({
                   minLength: 3,
                   highlight: true,
-              },
-              {
+            },
+            {
                   name: 'idtf',
                   source: function (query, cb) {
                       $('#search-input').addClass('search-processing');
-                      SCWeb.core.Server.findIdentifiersSubStr(query, function (data) {
+                      SCWeb.core.Server.findIdentifiersSubStr(query, function (list) {
                           keys = [];
 
-                          var addValues = function (key) {
-                              var list = data[key];
-                              if (list) {
-                                  for (idx in list) {
-                                      var value = list[idx]
-                                      keys.push({name: value[1], addr: value[0], group: key});
-                                  }
+                          if (list) {
+                              for (idx in list) {
+                                  var value = list[idx]
+                                  keys.push({name: value[1], addr: value[0]});
                               }
                           }
 
-                          addValues('sys');
-                          addValues('main');
-                          addValues('common');
-
                           cb(keys);
                           $('#search-input').removeClass('search-processing');
                       });
                   },
                   displayKey: 'name',
                   templates: {
                       suggestion: function (item) {
-
-                          //glyphicon glyphicon-globe
-                          var html = '';
-                          if (item.group === 'common') {
-                              return '<p class="sc-content">' + item.name + '</p>';
-                          } else {
-                              var cl = 'glyphicon glyphicon-user';
-                              if (item.group === 'sys') {
-                                  cl = 'glyphicon glyphicon-globe';
-                              }
-                              return '<p><span class="tt-suggestion-icon ' + cl + '"></span>' + item.name + '</p>';
-                          }
+                          var cl = 'glyphicon glyphicon-user';
+                          return '<p><span class="tt-suggestion-icon ' + cl + '"></span>' + item.name + '</p>';
```

Comment:
I suggest to remove the icon as we don't determine text type anymore
```suggestion
                          return '<p>' + item.name + '</p>';
```